rai/client_test.go

```diff
@@ -145,11 +65,13 @@ func findModel(models []Model, name string) *Model {
 
 // Test database management APIs.
 func TestDatabase(t *testing.T) {
-	client, err := newTestClient()
+	client, err := NewTestClient()
 	assert.Nil(t, err)
-	defer tearDown(client)
+	defer TearDownEngine(client, engineName)
+	defer TearDownDatabase(client, databaseName)
 
-	ensureEngine(t, client)
+	EnsureEngine(t, client, engineName)
```

Comment:
why was this refactored and why are we now exporting?

Reply:
For `ResultTable` I created a new package called `results`, the idea is to have a generic `test_utils` that could be used across different packages, previously `ensureDatabase` will create also an engine so I decoupled those since this is won't be required for some test use cases and made them available for other packages, finally this shouldn't be part of `client_test` I need to move it to `test_utils` :pray:

Reply:
if you have to share unit helpers, its an indication this maybe shouldnt be a separate package.

Reply:
yeah that's right, but in this case raicloud related units (like `NewTestClient`, `EnsureEngine`, ...) are required by all integration tests independently from their package